src/ExceptionTypes.f90

```diff
@@ -0,0 +1,370 @@
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+!                          Futility Development Group                          !
+!                             All rights reserved.                             !
+!                                                                              !
+! Futility is a jointly-maintained, open-source project between the University !
+! of Michigan and Oak Ridge National Laboratory.  The copyright and license    !
+! can be found in LICENSE.txt in the head directory of this repository.        !
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+!> @brief Utility module for defining exception types used by the
+!>  ExceptionHandler
+!>
+!> @author William Gurecky
+!>   @date 12/16/2019
```

Comment:
Delete the `@author` and `@date` lines.  Those probably exist throughout Futility but are no longer part of the code standard (because git)